src/tools/cephfs_mirror/PeerReplayer.cc

```diff
@@ -33,11 +33,15 @@ using namespace std;
 enum {
   l_cephfs_mirror_peer_replayer_first = 6000,
   l_cephfs_mirror_peer_replayer_snaps_synced,
+  l_cephfs_mirror_peer_replayer_sync_bytes,
```

Comment:
why change the enumerated value for this?

Reply:
oops, I just made them in order. Didn't thought about the value change back then. I'll revert it if needed?

Reply:
@vshankar Fixed